pac4j-http/src/main/java/org/pac4j/http/credentials/CredentialUtil.java

```diff
@@ -0,0 +1,61 @@
+package org.pac4j.http.credentials;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * This class contains utility methods related to credential encryption algorithms
+ * (MD5 for http digest)
+ *
+ * @author Mircea Carasel
+ */
+public class CredentialUtil {
```

Comment:
If you find some library, it would be an `optional` dependency (only required for Digest).